app/helpers/conservation_records_helper.rb

```diff
@@ -1,6 +1,13 @@
 # frozen_string_literal: true
 
 module ConservationRecordsHelper
+  def date_returned(record)
+    return if record.cost_return_report.blank?
+
+    returned_date = record.cost_return_report.returned_to_origin
+    return returned_date.to_date if returned_date.present?
```

Comment:
I'd like to see a test that covers lines 7-8 when the cost_return_report exists.